### ISSUE TITLE
ADS re-open previously closed tabs

### ISSUE THREAD

**Issue by drizztdourden08** [microsoft/azuredatastudio#14602]
- Azure Data Studio Version:

Steps to Reproduce:

Same issue as "ADS reopen closed files #4381"

1. Work in ADS
2. Close some tabs
3. Close ADS normally
4. Re-open ADS
4.1 If you left some tabs opened before closing, they'll open along with the one you already closed
4.2 if you left nothing opened and unsaved, only the one you closed will open

I am unsure what triggered that exactly to be honest. Right now, I have 3 tabs that keeps opening but not all of them are. The 3 tabs are all the same thing. the other tabs I close aren't re-opening so it's not happening all the time and to all tab. It's a specific issues that seem to be triggered by something specific.

![image](https://user-images.githubusercontent.com/36481532/110382411-dd03e800-8028-11eb-9112-fdf84f4804bf.png)

A screenshot of one of the tab. All of the 3 tabs that keeps re-opening even though I have closed them each time over a week ago are the last 3. The first 3 were not closed for this example and they appeared first followed by the 3 buggy one.

If I can provide anything else, let me know. I suppose this data is saved somewhere so I can provide it.

**Comment by drizztdourden08** [comment]
Also, these seems to be the only opened tab that don't ask to "save or discard". they just close straight no prompt.

**Comment by drizztdourden08** [comment]
Correction: It's 4 tabs that won't close. The first one in the screenshot was one of them. (called "SQLQuery_5")

Troubleshoot:
1. I closed everything
2. Opened ADS
3. Modified one of the buggy tabs
4. Closed ADS
5. Upon re-opening, ADS opened the same 4 buggy tabs again alons with the one I just modified. These 4 tabs are stuck somewhere.

I tried a few functions from F1 but couldn't find anything that would clear this.

![image](https://user-images.githubusercontent.com/36481532/110383868-ee4df400-802a-11eb-9099-5a19f6561d13.png)
